ddtrace/contrib/flask/patch.py

```diff
@@ -323,7 +322,9 @@ def traced_start_response(status_code, headers):
             config.flask,
             method=request.method,
             url=request.base_url,
-            query=compat.to_unicode(request.query_string),
+            # It's probably wrong to assume it's UTF-8, but there is no hint to know what the encoding of the query
+            # string is anyhow.
+            query=request.query_string.decode("utf-8", errors="replace"),
```

Comment:
is this a fix we should make in `compat.to_unicode`?

Reply:
I'm not sure. I'm not a fan of the `to_unicode` approach which tries to force everything, anyhow, without caring what the input type is, to convert to Unicode.
I'm more into a thoughtful approach on a case-by-case basis.